cinder/templates/etc/_volume-netapp.conf.tpl

```diff
@@ -0,0 +1,21 @@
+{{- define "volume_netapp_conf" -}}
+{{- $context := index . 0 -}}
```

Comment:
It seems we are not using the global $context in this template so we could avoid the tuple foo in this case, no?